README.md

```diff
@@ -1,20 +1,67 @@
+> [!IMPORTANT]
+> ThreadSharp is limited by the capabilities of the [Threads public API](https://developers.facebook.com/docs/threads/) and so not all features that you enjoy in the Threads app are available.
+
 ![ThreadSharp banner](https://github.com/itsWindows11/ThreadSharp/blob/main/assets/banner.png?raw=true)
+<h4 align="center"> A C# API wrapper for the Threads API.</h4>
+<p align="center">
+  <a style="text-decoration:none" href="https://github.com/itsWindows11/ThreadSharp/actions/workflows/nuget.yml">
+    <img src="https://github.com/itsWindows11/ThreadSharp/actions/workflows/nuget.yml/badge.svg" alt="CI Status" /></a>
+  <a style="text-decoration:none" href="https://discord.com/channels/714581497222398064/913496895006072843">
+    <img src="https://img.shields.io/badge/discuss-on%20discord-5865F2" alt="Discord" /></a>
+  <a style="text-decoration:none" href="https://www.nuget.org/packages/ThreadSharp">
+    <img src="https://img.shields.io/nuget/v/ThreadSharp" alt="Discord" /></a>
+</p>
 
-# <center>ThreadSharp</center>
-<center>A C# API wrapper for the Threads API.</center>
+## Documentation
 
-![NuGet Version](https://img.shields.io/nuget/v/ThreadSharp)
+You can browse the documentation at [itswindows11.github.io/ThreadSharp](https://itswindows11.github.io/ThreadSharp/docs/).
 
-## Documentation
+## Contributing
 
-You can browse the documentation at https://itswindows11.github.io/ThreadSharp.
+There are multiple ways to participate in the community:
+
+- Upvote popular feature requests
+- [Submit a new feature](https://github.com/itsWindows11/ThreadSharp/pulls)
+- [File bugs and feature requests](https://github.com/itsWindows11/ThreadSharp/issues/new/choose).
+- Review source [code changes](https://github.com/itsWindows11/ThreadSharp/commits)
 
 ## Usage
 
+> [!NOTE]
```

Comment:
It would make more sense if it's put below the code snippet, where the need for authentication in the first place is defined.